src/apps/vpn/telemetry/pings.yaml

```diff
@@ -16,6 +16,15 @@ vpnsession:
     session-based metrics, etc.
   include_client_id: true
   send_if_empty: false
+  reasons: 
+    flush: |
+      Safety measure for when the vpn session does not end in an expected way, sent right before the start ping
+    start: |
+      When the vpn session starts
+    end: |
+      When the vpn session ends
```

Comment:
Same comment on documentation. I'd also add any caveats, such as "A VPN session never ends on a timeout" or "A VPN session doesn't end unless a user explicit turns the VPN off or force kills the VPN process". (Not sure these example are accurate, they are just examples).